Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -277,6 +277,59 @@ public void Roslyn_Settings_ValidSetup_NonLegacyServer_MergeSettings()
             AssertWarningsAreNotTreatedAsErrorsNorIgnored(result);
         }
 
+        [TestMethod]
```

Comment:
Integration test that actually executes against a project that doesn't specify a language.

I considered adding a Java IT using the specific project type reported in the bug (_.sfproj_), but the test machine would need to have an additional Azure SDK installed.